docs/install/from_source.rst

```diff
@@ -73,6 +73,16 @@ linux operating systems, execute (in a terminal):
     sudo apt-get update
     sudo apt-get install -y python3 python3-dev python3-setuptools gcc libtinfo-dev zlib1g-dev build-essential cmake libedit-dev libxml2-dev
 
+On macOS, whether using an Intel or M1 Mac we can use homebrew. Make sure to follow the post installation steps for each of the below commaands.
```

Comment:
nit: "M1 Mac, "

Reply:
I would go so far as to say "only Homebrew installations have been tested."

Reply:
Done.